pkgs/development/python-modules/ahocorasick-rs/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  rustPlatform,
+  typing-extensions,
+  pytestCheckHook,
+  pyahocorasick,
+  hypothesis,
+  pytest-benchmark,
+}:
+
+buildPythonPackage rec {
+  pname = "ahocorasick-rs";
+  version = "0.22.0";
+
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "ahocorasick_rs";
+    sha256 = "sha256-lzRwODlJlymMSih3CqNIeR+HrUbgVhroM1JuHFfW848=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-CIt/ChNcoqKln6PgeTGp9pfmIWlJj+c5SCPtBhsnT6U=";
+  };
+
+  nativeBuildInputs = with rustPlatform; [
+    maturinBuildHook
+    cargoSetupHook
+    typing-extensions
```

Comment:
If anywhere this should be in `dependencies`:
https://github.com/G-Research/ahocorasick_rs/blob/034e3f67e12198c08137bb9fb3153cb01cf5da31/pyproject.toml#L9-L10